Jefiozie/scully
# playwright

- commitje
- add logging
- install chromium
- chore(scully-plugin-critical-css): update critical to 4.0.1 (#1423)
- add some extra things for playwright
- remove logging
- chore: remove puppeteer completly
- chore: update config to use normal config
- chore: make config based on scullyconfig
- use 1 thread
- Update playwrightUtils.ts
- ci: add playwright deps for test`
- ci: run with args
- lets work
- ci: add no cache
- ci: install deps
- add a test flow
- ci: add sample
- ci fix
- ci: now then
- ci: fix
- helpp
- ci: 1
- ci: test
- ci: add
- remove duplicate install and ci flag
- add debug flags
- Update check.yml
- remove env
- define run
- chore:  replace with normal playwright
- update package
- default to chromium
- remove browser download env
- use install instead of ci
- with ngcc
- without ngcc
- npm ci
- npm install
- with npm install and debug
- add caching
- ci: add needs
- back to npm install
- add debug log
- ci: log lever to error
- ci: remove args and set debug level
- add caching
- comment options
- add options
- use chromium
- add logging
- log browser type
-  use chromium direct
-  remove cache
- emove needs
- helppppp
- remove debug log
- help 2
- help 3
- no cache
- add samples
- add debug
- add sandbox arg
- ci: remove all args
- with puppet options
- this
-  with logline
- better loggingp
- log html
- log HTML on error

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit

## PR Type

What kind of change does this PR introduce?




## What is the current behavior?



Issue Number: N/A

## What is the new behavior?

## Does this PR introduce a breaking change?




## Other information
